### PR TITLE
mvcc: do not hash consistent index

### PR DESCRIPTION
**Pull request by xiang90**
fix https://github.com/coreos/etcd/issues/5791

### CODE REVIEW

File: mvcc/backend/backend.go [hunk 1]
```diff
@@ -144,7 +144,12 @@ func (b *backend) Snapshot() Snapshot {
 	return &snapshot{tx}
 }
 
-func (b *backend) Hash() (uint32, error) {
+type IngoreKey struct {
```
Comment (heyitsanthony) [inline mvcc/backend/backend.go]: s/Ingore/Ignore

Comment (xiang90) [inline mvcc/backend/backend.go]: fixed.

**Comment by heyitsanthony** [comment]
lgtm

**Comment by gyuho** [comment]
lgtm. thanks! I will redeploy with this patch after merge.

**Pull request merged**